RocketChat/Rocket.Chat
# feat: Increase the limit of members that can be added using the autojoin feature in a team's channel

## Proposed changes (including videos or screenshots)
There existed an implicit limit of 50 members that would be included whenever a a channel with auto join was created inside a team, this task ties the limit to the API_Users_Limit setting, so it is configurable.



## Issue(s)


## Steps to test or reproduce

This task requires a team with over 1000 members, so all of these members should be created and associated with the team
1. Create a channel from the team
2. Open the kebab-menu and mark the auto join box
All the members should've been added correctly
3. Add one more user to the team
4. Create a new channel in the team
5. Mark the auto join box
Now the new user should not have been added since there is a hard limit of 1000 members


## Further comments

[SUP-34]

[SUP-34]: https://rocketchat.atlassian.net/browse/SUP-34?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
